android/UtilsPlugin.java

```diff
@@ -241,6 +259,39 @@ public void onNewIntent(Intent gameIntent) {
 		}
 	}
 
+	public void openAppSettings() {
+		Uri packageURI = Uri.parse("package:" + _context.getPackageName());
+		Intent intent = new Intent("android.settings.APPLICATION_DETAILS_SETTINGS", packageURI);
+		intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?

got above error and the searched found ,need to add flag